content/applications/marketing/events/event_templates.rst

```diff
@@ -0,0 +1,217 @@
+
+.. seealso::
+   To learn more about event forms, refer to the :doc:`create_events` documentation.
+
+Create event template
+=====================
+
+There are two ways users can create and configure an event template in Odoo *Events*.
+
+Either navigate to :menuselection:`Events app --> Configuration --> Event Templates`, and click the
+:guilabel:`New` button in the upper-left corner. Doing so reveals a blank event template form that
+can be customized in a number of different ways.
+
+Or, on an event form, start typing the name of a new event template in the *Template* field, and
+click :guilabel:`Create and edit...` from the resulting drop-down menu. Doing so reveals a *Create
+Template* pop-up window, featuring all the same configurable fields and elements found on a standard
+event template form.
+
+.. note::
+   Clicking :guilabel:`Create "[template name]"` from the resulting drop-down menu, via the
+   *Template* field on an event form creates the event template in the database, but does **not**
+   present the user with the *Create Template* pop-up window.
+
+   The event template would have to be modified, by selecting it on the *Event Templates* page
+   (:menuselection:`Events app --> Configuration --> Event Templates`).
+
+Event template form
+-------------------
+
+All the fields on a standard :guilabel:`Event Template` form are *also* on the *Create Template*
+pop-up window, accessible via the *Template* field on an event form.
+
+.. image:: event_templates/event-template-form.png
+   :align: center
+   :alt: A standard event template form in the Odoo Events application.
+
+Start by providing the event template with a name in the :guilabel:`Event Template` field, located
+at the top of the form.
+
+Beneath that field, there is a series of selectable checkboxes, all of which are related to how the
+event menu will be displayed on the event web page.
+
+- :guilabel:`Website Submenu`: Odoo presents an event-specific submenu of selectable options that
+  are related to that specific event. When this checkbox is ticked, every other checkbox in this
+  series is automatically ticked, as well. Users can then choose to untick (deselect) any of the
+  checkbox options, as desired.
+- :guilabel:`Tracks Menu Item`: Adds a menu item to navigate to a page displaying all planned tracks
+  for the event.
+- :guilabel:`Track Proposals Menu Item`: adds a menu item to navigate to a page, in which visitors
+  can fill out a customized form to propose a track (talk, lecture, presentation, etc.) to happen
+  during the event.
+- :guilabel:`Booth Menu Item`: adds a menu item that takes visitors to a separate page, where an
+  event booth can be purchased. Event booths can be customized and configured in the
+  :guilabel:`Booths` tab of the event template form, from the *Booth Categories* page
+  (:menuselection:`Events app --> Configuration --> Booth Categories`). But first, users **must**
```

Comment:
Suggestion around presentation:
The "...**must** create a booth product" part could be an inline admonition block (note, important) inside the list, to further visually highlight the importance of making a product first.